dashboard/lib/sign_up_tracking.rb

```diff
@@ -4,8 +4,8 @@
 module SignUpTracking
   STUDY_NAME = 'account-sign-up-v2'
   NOT_IN_STUDY_GROUP = 'not-in-study'
-  CONTROL_GROUP = 'control'
-  NEW_SIGN_UP_GROUP = 'experiment-v2'
+  CONTROL_GROUP = 'control-v2'
+  NEW_SIGN_UP_GROUP = 'experiment-v3'
```

Comment:
I wonder if it'd be less confusing to just keep these in lockstep going forward: `control-v3` and `experiment-v3`.

Reply:
good call 👍